data/en/1_0005_updated.json

```diff
@@ -0,0 +1,2168 @@
+                                    "q_code": "94f1",
+                                    "value": "more overtime"
+                                },
+                                {
+                                    "label": "less overtime",
+                                    "q_code": "94f2",
+                                    "value": "less overtime"
+                                },
+                                {
+                                    "label": "pay increase",
+                                    "q_code": "95f",
+                                    "value": "pay increase"
+                                },
+                                {
+                                    "label": "industrial action",
+                                    "q_code": "96f",
+                                    "value": "industrial action"
+                                },
+                                {
+                                    "label": "other",
+                                    "q_code": "97f",
+                                    "value": "other"
+                                }
+                            ],
+                            "type": "Checkbox"
+                        }],
+                        "id": "fortnightly-pay-significant-changes-options-question",
+                        "title": "What were the reasons for the changes to  the <em>total pay or number</em> of <em>fortnightly</em> paid employees?",
+                        "type": "General"
+                    }],
+                    "routing_rules": [{
+                            "goto": {
+                                "block": "fortnightly-pay-significant-changes-pay-rates-increase",
+                                "when": [{
+                                    "id": "fortnightly-pay-significant-changes-options-answer",
+                                    "condition": "contains",
+                                    "value": "pay increase"
+                                }]
+                            }
+                        },
+                        {
+                            "goto": {
+                                "block": "fortnightly-pay-significant-changes-other-specify"
+                            }
+                        }
+                    ]
+                },
+                {
+                    "id": "fortnightly-pay-significant-changes-pay-rates-increase",
+                    "questions": [{
+                        "answers": [{
+                                "id": "fortnightly-pay-significant-changes-pay-rates-increase-percent-increase-answer",
+                                "label": "Percentage increase of new pay rates",
+                                "mandatory": false,
+                                "q_code": "100f",
+                                "type": "Percentage",
+                                "max_value": {
+                                    "value": 100
```

Comment:
Should this be limited to 100? a % increase can exceed 100.

Reply:
@jonnyshaw89 Yes technically your right but its a business requirement to have it capped at 100%